main/postgresql/APKBUILD

```diff
@@ -119,6 +122,46 @@ contrib() {
 	rmdir -p "$subpkgdir"/usr/share/doc/postgresql || true
 }
 
+plperl() {
+	depends="postgresql perl"
```

Comment:
Dependency on perl is automatically detected by abuild via dynamic links, so it’s not needed (and desirable) to specify it here. The same applies even for plpython and pltcl.

```
>>> postgresql-plperl*: Tracing dependencies...
	postgresql
	perl
	busybox
	so:libc.musl-x86_64.so.1
-->	so:libperl.so  <--
```